sensor/DeviceOfflineSensor.js

```diff
@@ -43,12 +43,19 @@ class DeviceOfflineSensor extends Sensor {
 
     log.debug("Start to check device activity...");
     const hostEntries = await hostTool.getAllMACEntries();
-    hostEntries.forEach((host) => {
+    hostEntries.forEach(async (host) => {
       if (host) {
+        let customizedOfflineIdle;
+        const policy = await hostTool.loadDevicePolicyByMAC(host.mac);
+        if (policy && policy["customizedOfflineIdle"]) {
```

Comment:
```
pi@firewalla:~ (MyGoldWalla) $ rc hget policy:system qos
"{\"state\":true,\"qdisc\":\"fq_codel\"}"
```

* use a new key other than hash key deviceOffline for compatibility
* improve isFeatureEnabled function to let it support state subkey as well as boolean (true/false)
  * deviceOffline: true
  * or deviceOffline: '{"state": true}'